src/objective-c/generated_libraries/RemoteTestClient/RemoteTest.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.prepare_command = <<-CMD
     BINDIR=../../../../bins/$CONFIG
     PROTOC=$BINDIR/protobuf/protoc
```

Comment:
Yeah, sorry for that :-/ This file evolved to be used exclusively by the automatic tests (running `./tools/run_tests/run_tests.py -l objc` from the root of the repo). Those will `make plugins` and set `$CONFIG` before doing `pod install`, making the podspec work.

As it became inappropriate for a sample, I added a version of it that just uses the installed protoc + plugin [inside the examples directory](https://github.com/grpc/grpc/tree/master/src/objective-c/examples/RemoteTestClient). The Podfile of this sample should just point to that one (as the Podfile for the SwiftSample does). And ideally we should move `generated_libraries/RemoteTestClient/*` into `tests/RemoteTestClient/*` to eliminate confusion. (And also eliminate `generated_libraries/RouteGuideClient`, which isn't used anymore.)

Do you want to do that? Otherwise I'm happy to send a PR to your branch with the changes.